shared/common-adapters/user-proofs.native.js

```diff
@@ -31,7 +30,7 @@ function MissingProofRow ({missingProof, style}: {missingProof: MissingProof, st
           </Box>
         </Box>
         <Box style={styleStatusIconContainer}>
-          <Icon type={'iconfont-proof-placeholder'} style={{...styleStatusIcon('iconfont-proof-placeholder'), color: missingColor}} />
+          <Icon type={'iconfont-proof-placeholder'} style={{...styleStatusIcon('iconfont-proof-placeholder'), color: globalColors.black_10}} />
```

Comment:
Maybe want to change `missingColor = globalColors.black_10` or is it specific to just here.